enterprise-suite/es-monitor-api/default-monitors.json

```diff
@@ -104,6 +104,24 @@
           "description": "{{$labels.instance}} has anomalous scrape_duration_seconds"
         }
       },
+      "es_monitor_api_errors": {
```

Comment:
"lightbend" instead of "es" ?  what do you think?

Reply:
how about 
lb_console_monitor_api_errors